echopype/preprocess/api.py

```diff
@@ -136,9 +137,17 @@ def _freq_MVBS(ds, rint, pbin):
             "binning_mode": "physical units",
             "range_meter_interval": str(range_meter_bin) + "m",
             "ping_time_interval": ping_time_bin,
+            "actual_range": [
+                round(float(ds_MVBS["Sv"].min().values), 2),
+                round(float(ds_MVBS["Sv"].max().values), 2),
+            ],
         }
     )
 
+    prov_dict = echopype_prov_attrs(process_type="processing", source_files=None)
```

Comment:
It seems weird to me that `source_files=None` here. But it is true that the source of the Sv files users open up is not accessible by use. 

I can see that if we implement this as part of a pipeline we can add this info there (the same chain of provenance discussion that is _not now_), but I wonder if there's a way for us to remind users to add this. Perhaps can only make it as explicit as possible in the docs?

Reply:
> It seems weird to me that source_files=None here. But it is true that the source of the Sv files users open up is not accessible by use.

Exactly. See the contortion I implemented for this provenance info for calibration functions in `calibrate/api.py`:
```python
    add_attrs(cal_type, cal_ds)
    # Provenance source files may originate from raw files (echodata.source_files)
    # or converted files (echodata.converted_raw_path)
    if echodata.source_file is not None:
        source_file = echodata.source_file
    elif echodata.converted_raw_path is not None:
        source_file = echodata.converted_raw_path
    else:
        source_file = "SOURCE FILE NOT IDENTIFIED"
    prov_dict = echopype_prov_attrs(process_type="processing", source_files=source_file)
```

For downstream (pre)processing functions, it becomes harder to track unambiguously.

> I can see that if we implement this as part of a pipeline we can add this info there (the same chain of provenance discussion that is not now), but I wonder if there's a way for us to remind users to add this. Perhaps can only make it as explicit as possible in the docs?

Reminding users of adding desirable but optional attributes is tricky business. I think that should all be part of post-0.6.0 provenance discussions. Not now.

Reply:
Yes, definitely. Let's merge this and worry about that later.